src/test/java/com/hubspot/jinjava/lib/tag/ForTagTest.java

```diff
@@ -193,6 +195,18 @@ public void testForLoopRangeWithStringsWithSpaces() {
       assertEquals("a b", rendered);
   }
 
+  @Test
+  public void testForLoopWithDates() {
+    Map<String, Object> context = Maps.newHashMap();
+    Date testDate = new Date();
+    context.put("the_list", Lists.newArrayList(new Date()));
```

Comment:
I think you want to use `testDate` when initializing `the_list` right?

Reply:
Yep, good catch. I guess this test worked anyway, because PyishDates aren't that precise? (`toString` only prints out to seconds, not millis).